modules/internal/ChapelComplex_forDocs.chpl

```diff
@@ -55,29 +74,30 @@ module ChapelComplex_forDocs {
   */
```

Comment:
Let's make the transition better above in the "It is defined". Maybe "``complex`` is defined as if it were a record..."